source/test/TestApp/GDK/MicrosoftGame.Config

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Game configVersion="0">
+  <Identity Name="TestClientApp" Publisher="CN=Publisher" Version="1.0.0.0"/>
+
+  <ExecutableList>
+    <Executable Name="TestClientApp.exe" Id="Game"/>
+  </ExecutableList>
+

```

Comment:
doesn't have xbl content so can't get xtoken.  you'll need this eventually so feel free to add later but fyi.

Reply:
Not currently needed.  Will add when we add GDK specific login tests